### PR TITLE
Updated Playfab Dependencies

### PR DESCRIPTION
**Pull request by SebastianPD-XB**
Updated PlayFab Party to 1.7.6
Updated PlayFab Multiplayer to 1.3.0

Purpose is to have a more up to date sample with the newest features that anyone can use.

### CODE REVIEW

**Comment by SebastianPD-XB** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by SebastianPD-XB** [comment]
@microsoft-github-policy-service agree

File: UnityRumble-PlayFabSteam/UnityRumble_Desktop/Assets/PlayFabMultiplayerSDK/PlayFabMultiplayerSDKDotNet.csproj.meta [hunk 1]
```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
+guid: 467859c7876ccd94ab111f8a7462fad0
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant:
```
Comment (Vicodex) [inline UnityRumble-PlayFabSteam/UnityRumble_Desktop/Assets/PlayFabMultiplayerSDK/PlayFabMultiplayerSDKDotNet.csproj.meta:7]: Please remove this file

**Comment by Vicodex** [comment]
>   assetBundleVariant: 

Please remove this file

---
Refers to: UnityRumble-PlayFabXboxLiveGDK/UnityRumble_Desktop/Assets/PlayFabMultiplayerSDK/PlayFabMultiplayerSDKDotNet.csproj.meta:7 in f4a2246. [](commit_id = f4a224643e3c23818a4ef61959b2ea591b91da86, deletion_comment = False)

**Comment by Vicodex** [comment]
We may need to update dependencies in UnityRumble-PlayFabXboxLiveGDK/UnityRumble_Desktop/Assets

**Review by Vicodex — CHANGES_REQUESTED** [review]
:clock1:

**Review by Vicodex — APPROVED** [review]

**Pull request merged**